test/migrations/credential-username-userid-transform.js

```diff
@@ -0,0 +1,55 @@
+const migrate = require('migrate');
+const tmp = require('tmp');
+const db = require('../../lib/db')();
+const { assert } = require('chai');
+const userService = require('../../lib/services/consumers/user.service');
+const credentialService = require('../../lib/services/credentials/credential.service');
+
+describe('Migrations', () => {
+  describe('Username -> UserID credential transform', () => {
+    let tmpFile;
+    let userId;
+    let oldCredential;
+    before('I insert some credentials by user ID', () => {
```

Comment:
I guess the name should be 
`I insert some credentials by username`

Reply:
Yeah!